crates/ra_ide/src/completion/completion_item.rs

```diff
@@ -285,6 +304,51 @@ impl Builder {
         self.deprecated = Some(deprecated);
         self
     }
+    #[allow(unused)]
```

Comment:
why is this allow unused?

Reply:
Because at this time I don't really use it. I just wanted to keep the same way than it was done with other fields. To stay consistent